code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -32,7 +32,7 @@
 /obj/item/rig/industrial
 	name = "industrial suit control module"
 	suit_type = "industrial hardsuit" //Credit for sprites go to Près de l'oiseau over at Eris
-	desc = "A heavy, powerful rig used by construction crews and mining corporations. The soft plating can protect the wearer from unexpected hazards, but is not reliable in prolonged combat."
+	desc = "A heavy, powerful rig used by construction crew and mining corporations. The soft plating can protect the wearer from unexpected hazards, but is not reliable in prolonged combat."
```

Comment:
Please revert. Corporations is plural and crews should remain plural.

Reply:
This is not grammatically correct. Crew is already a nebulous term that is both singular and plural. "Crew" is the plural of "crew".

In American and British English standard, unless I've missed something, the word "crews" is an unrecognized slang.

Reply:
Merriam-Webster suggests crews is a correct form, I'd have to dig deeper. This language sucks.

Reply:
Crews is correct, as it implies the rig is used by multiple groups of construction workers.

Reply:
All I'm commenting is that my contempt for American English grows, I'll adamantly disagree on this in spirit. Changes made.